### ISSUE TITLE
Details regarding equity value and profit in trades is not always clear.

### ISSUE THREAD

**Issue by gmit22** [gmit22/pe#5]
Even though the profit/loss = 0 currently, on typing the command `wallet` it shows some loss. This may be due to api limit, however the user should not see incorrect information at any time. This differs from the behaviour, and would create some confusion.

It is confirmed by the display of 'view' command above it.
![Capture.PNG](https://raw.githubusercontent.com/gmit22/pe/main/files/75170419-6c28-48e4-843e-1f0d26f33d6f.PNG)
